internal/access/access_key.go

```diff
@@ -27,9 +34,15 @@ func ListAccessKeys(c *gin.Context, identityID uid.ID, name string, showExpired
 }
 
 func CreateAccessKey(c *gin.Context, accessKey *models.AccessKey) (body string, err error) {
-	db, err := RequireInfraRole(c, models.InfraAdminRole)
-	if err != nil {
-		return "", HandleAuthErr(err, "access key", "create", models.InfraAdminRole)
+	var db data.GormTxn
+
+	if accessKey.IssuedFor == GetRequestContext(c).Authenticated.User.ID {
+		db = getDB(c) // can create access keys for yourself.
+	} else {
+		db, err = RequireInfraRole(c, models.InfraAdminRole)
+		if err != nil {
+			return "", HandleAuthErr(err, "access key", "create", models.InfraAdminRole)
```

Comment:
```suggestion
			return "", HandleAuthErr(err, "access keys", "create", models.InfraAdminRole)
```
Nit: Not related to this, change.

Reply:
Right now I think we always use the singular form for `create` since it's creating a single key, and we use the plural for `list`, since it generally returns multiple.  I think get and delete likely use the singular as well.

I think it's fine if we want to change everything to plural, but maybe we should do that everywhere in a separate PR?